rise-tools/rise-tools
# feat: add kit-haptics, add validation to other kit action packages, better validation errors

Fixes #92 

Validation:
- fix an error when validating action would always throw an error
- add better error message for invalid payloads
- add validators for all actions (in expo router, toast and haptics). In the future, this will be less boilerplate when we use `Typia` (can infer validator from TS type)
- Ensure validator (such as Zod) returns expected shape from `validate` function
- Slightly more convenient `ActionsDefinition` helper
